junit-platform-engine/src/main/java/org/junit/platform/engine/support/descriptor/FilePosition.java

```diff
@@ -52,6 +54,50 @@ public static FilePosition from(int line, int column) {
 		return new FilePosition(line, column);
 	}
 
+	/**
+	 * Create an optional {@code FilePosition} parsing the supplied
+	 * {@code query} string.
+	 *
+	 * <p>Examples of valid {@code query} strings:
+	 * <ul>
+	 *     <li>{@code "line=23"}</li>
+	 *     <li>{@code "line=23&column=42"}</li>
+	 * </ul>
+	 *
+	 * @param query the query string; may be {@code null}
+	 * @since 1.3
```

Comment:
missing `@return` tag

Reply:
✔️ (adding two more to the other `FilePosition::from` factory methods)

Reply:
Note: all concrete `TestSource` implementation factory methods lack `@return` tags.

Reply:
Aha. Then I guess we should likely address that in a separate commit outside the scope of this PR.